### ISSUE TITLE
Inline analog clock component

### ISSUE THREAD

**Issue by kitsuyui** [kitsuyui/react-playground#41]
I want an inline analog clock component.

The goal is not to provide a "small" clock with opinionated sizing or styling.
This repository should provide reusable, style-agnostic React components, and consumers should control presentation by themselves.

What I want here is an analog clock that behaves more like inline text or an icon:

- it can be placed naturally inside a line of text
- it can be used in places such as the end of a menu bar
- it should be stylable in the same way as a `span`
- it should inherit typography and color from surrounding content
- it should avoid imposing fixed width, fixed font size, margin, or other presentational styling

In other words, this is not a request for a "small analog clock" component.
It is a request for an inline analog clock component that is compatible with ordinary HTML/CSS usage.
